### ISSUE TITLE
possible additional feature - a 'step' button?

### ISSUE THREAD

**Issue by wrincewind** [dwilliamson/dcpu16.js#5]
just a button you can press to execute one line[or more likely, one cycle] at a time. would be useful, especially for debugging loops and the like.

**Comment by dwilliamson** [comment]
I can easily add a button for this, but doesn't F10 work for you? That does the stepping...

**Comment by wrincewind** [comment]
honestly, i'm not sure. using f10 doesn't seem to affect the program... i  can't see what line it's currently on. does it highlight the current line, or what?

**Comment by dwilliamson** [comment]
Yes, when the program breaks (insert a "BRK" instruction or press F9) the current line is marked with a red background.
